VirtoCommerce.CoreModule.Web/Controllers/Api/StoreFrontSecurityController.cs

```diff
@@ -331,5 +331,40 @@ public class StorefrontSecurityController : ApiController
             }
             return email;
         }
+
+        /// <summary>
+        /// Send Forgot UserName notification for sign in
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="storeName"></param>
+        /// <param name="language"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("user/sendusername")]
+        [ResponseType(typeof(void))]
+        public async Task<IHttpActionResult> SendUserNameNotification(string userId, string storeName, string language)
```

Comment:
The method should be called  `RememberUserName(string userId, string storeId, string language)`